tests/pytest_tests/system_tests/tests_launch/test_launch.py

```diff
@@ -128,3 +129,12 @@ def test_launch_multi_run_context(
 
         assert run1.id == "test"
         assert run2.id != "test"
+
+
+def test_launch_get_project_queue_error(user):
+    proj = "projectq32e"
+    api = InternalApi()
+    with pytest.raises(CommError) as e:
```

Comment:
you can use the `match` argument in `pytest.raises`. It replaces the assertion you have in the end of the test.